design/design-doc-stacks.md

```diff
@@ -1,15 +1,21 @@
-# Crossplane Extensions
+# Crossplane Stacks
 * Owner: Jared Watts (@jbw976)
 * Reviewers: Crossplane Maintainers
-* Status: Accepted, revision 1.0
+* Status: Accepted, revision 1.1
 
-This document aims to provide details about the experience and implementation for Crossplane “extensions”, which can add new functionality/support, types, and controllers to Crossplane.
+This document aims to provide details about the experience and implementation for Crossplane “Stacks”, which can add new functionality/support, types, and controllers to Crossplane.
```

Comment:
Should Stacks remain in quotations given it's almost as much a brand as a concept?

Reply:
I considered dropping the quotes too.  Consider them dropped.